js/AddNewItem.js

```diff
@@ -0,0 +1,51 @@
+import { renewStrong } from "./ControlTodoButton.js";
+import { ajaxPostFunctions } from "./AjaxPost.js"
+
+const getWork = document.querySelector(".new-todo"); 
+export const todoList = document.querySelector(".todo-list"); 
+
+export function initAddNewItem() {
+  getWork.addEventListener("keypress", AddNewList);
+}
+
+export function AddNewList(e) {
+  if (e.key !== "Enter") return;
+  if(e.target.value.length===1){
+    alert("두 글자 이상 입력해주세요!");
+    return;
+  }
```

Comment:
```suggestion
  const MINIMUM_USER_NAME_LENGTH = 2;

  if(e.target.value.length < 2){
    alert(`${MINIMUM_USER_NAME_LENGTH}글자 이상 입력해주세요!`);
    return;
  }
```
이런 식으로 미션에서 주어진 특정 조건을 상수(constant)로 분리하시면 가독성도 좋아지고 후에 조건을 변경하게 될 경우에도 보다 손쉽게 수정이 가능할거 같습니다!

Reply:
조건을 상수로 분리하는 것은 프로그래밍 공부를 하던 이전에도 몇번 지적받았던 사항인데 역시 습관화 되지 않으면 자꾸 까먹는거같습니다 ㅠ
review#2 에 반영하여 수정하였습니다. 감사합니다!